### PR TITLE
Removed community it from dashboard

### PR DESCRIPTION
**Pull request by Mte90**
Now there is MCWS so we don't need anymore to track them in the dashboard.

### CODE REVIEW

**Comment by Mte90** [comment]
I will do it asap

File: remo/remozilla/tasks.py [hunk 1]
```diff
@@ -46,7 +45,8 @@ def parse_bugzilla_time(time):
 
 @app.task
 @transaction.atomic
-def fetch_bugs(components=COMPONENTS, days=None):
+def fetch_bugs(components=
```
Comment (akatsoulas) [inline remo/remozilla/tasks.py]: Can you revert this change back? There is no need to break the line here.
